### PR TITLE
Ignore disruption by default an match api parameter names in ComponentReadiness UI

### PR DESCRIPTION
**Pull request by DennisPeriquet**
[TRT-1068](https://issues.redhat.com//browse/TRT-1068)

This is dependent on https://github.com/openshift/sippy/pull/1003
I tested this against PR1003 and traversed all 5 pages and got no errors from the API.

### CODE REVIEW

File: sippy-ng/src/component_readiness/CompReadyUtils.js [hunk 1]
```diff
@@ -282,8 +282,8 @@ export function getUpdatedUrlParts(
     confidence: confidence,
     pity: pity,
     minFail: minFail,
-    ignoreDisruption: ignoreDisruption,
-    ignoreMissing: ignoreMissing,
+    ignore_disruption: ignoreDisruption,
+    ignore_missing: ignoreMissing,
```
Comment (stbenjam) [inline sippy-ng/src/component_readiness/CompReadyUtils.js:286]: Why the mismatch between camel and snake case?

Comment (DennisPeriquet) [reply sippy-ng/src/component_readiness/CompReadyUtils.js:286]: the left side is used for the api call; the right side is the value I'm using.

Comment (DennisPeriquet) [reply sippy-ng/src/component_readiness/CompReadyUtils.js:286]: the left side is used for the api call; the right side is the value I'm using.

Comment (stbenjam) [reply sippy-ng/src/component_readiness/CompReadyUtils.js:286]: Why? The rest of the values here are camel case above these two lines

**Pull request merged**